packages/catlog/src/dbl/theory.rs

```diff
@@ -330,8 +316,8 @@ pub enum TabMorOp<V, E> {
     /// Identity operation on a morphism type.
     Id(TabMorType<V, E>),
 
-    /// Hom operation on an object operation.
-    Hom(TabObOp<V, E>),
+    /// Mor operation on an object operation.
+    Mor(TabObOp<V, E>),
```

Comment:
How are going to avoid confusion here since we already have an `Id` in this enum?